scripts/migrate-document-groups.js

```
@@ -211,39 +231,168 @@ async function previewBackfillCmUnassigned() {
+    try {
+      await db.doc(MAINTENANCE_FLAGS_DOC_PATH).set(
+        { groupAggregationGateOpen: true, gateOpenedAt: admin.firestore.FieldValue.serverTimestamp() },
+        { merge: true }
+      );
+    } catch (gateError) {
+      console.error(
+        '❌ メンテナンスゲートの再開に失敗しました。ゲートが閉じたままの可能性があります。' +
+        'system/maintenanceFlags の groupAggregationGateOpen を手動で true に戻してください: ',
+        gateError
+      );
+      throw gateError;
```

Biome (2.5.3)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)